File: eDental-flutter-main/lib/models/treatment.dart
import 'dart:typed_data';
import 'package:dart_json_mapper/dart_json_mapper.dart';

@jsonSerializable
class Treatment {
  int id;
  String name;
  double price;
  int timeRequiredInMinutes;
  ByteBuffer? image;

  Treatment(
    this.id,
    this.name,
    this.price,
    this.timeRequiredInMinutes,
    this.image,
  );
}

File: edentalbackend-main/DentalOffice.Entities/dbEntities/Treatment.cs
namespace DentalOffice.Entities.dbEntities
{
    public class Treatment
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal Price { get; set; }
        public int TimeRequiredInMinutes { get; set; }
        public byte[]? Image { get; set; }
    }
}
